integration_test/ops_agent_test.go

```diff
@@ -222,6 +222,13 @@ func installOpsAgent(ctx context.Context, logger *logging.DirectoryLogger, vm *g
 		}
 		return nil
 	}
+	// Removing flaky rhel-7 repositories due to b/265341502
+	if gce.IsRHEL7(vm.Platform) {
```

Comment:
we should do this somewhere more general. I suggest here: https://github.com/GoogleCloudPlatform/ops-agent/blob/a200ece61d0bbba1d7e21ae6c54004a69e4b9799/integration_test/gce/gce_testing.go#L1159

Then the repos will be disabled exactly once, at VM startup, before we start using it.

Reply:
also if you do move it there, let's make isRhel7() private (lowercase) because I don't think we're going to want to use it outside of this file. We can always make it public later.

if you want to keep it as public (uppercase), that's OK with me but in general I prefer to only make things public if we have to.

Reply:
Addressed in https://github.com/GoogleCloudPlatform/ops-agent/pull/1062/commits/585ed6fe32bd2c38005fd7f344bcae7993f4d980 , made method private.